integration_tests/web/test_conversations_connect.py

```diff
@@ -76,6 +80,19 @@ def test_sync(self):
                 receiver.conversations_approveSharedInvite,
                 invite_id=invite["invite_id"],
             )
+
+            sender_approval = sender.conversations_acceptSharedInvite(invite_id=invite["invite_id"], team_id=connect_team_id)
+            self.assertIsNone(sender_approval["error"])
+
```

Comment:
@filmaj @WilliamBergamin To make this test function, perhaps you need to call conversations.approveSharedInvite here. Without that, the followng calls will result in channel_not_found error